libs/application/templates/health-insurance/src/forms/HealthInsuranceForm.ts

```diff
@@ -43,7 +43,7 @@ export const HealthInsuranceForm: Form = buildForm({
         buildExternalDataProvider({
           title: m.externalDataTitle,
           id: 'approveExternalData',
-          subTitle: m.externalDataSubtitle,
```

Comment:
Should perhaps remove this field from the message file (and contentful) if we are not using it anymore?

Reply:
Ohh realised it is used on the "background" page